bin/webpack

```diff
@@ -7,9 +7,12 @@ require "pathname"
 ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
   Pathname.new(__FILE__).realpath)
 
-require "rubygems"
 require "bundler/setup"
 
 require "webpacker"
 require "webpacker/webpack_runner"
-Webpacker::WebpackRunner.run(ARGV)
+
+APP_ROOT = File.expand_path("..", __dir__)
+Dir.chdir(APP_ROOT) do
+  Webpacker::WebpackRunner.run(ARGV)
+end
```

Comment:
Not sure I grok what's happening here yet.

Reply:
I just updated the binstubs. `bin/rails webpacker:binstubs`

Reply:
It was an attempt to fix the dev server issues but figured might as well keep the change.